### ISSUE TITLE
[feature request] accept mulitple language JSON file

### ISSUE THREAD

**Issue by tsl143** [ryandrewjohnson/react-localize-redux#14]
Right now we have to give one JSON file with multiple language strings as an array.
`"title": [`
`      "Hello from Info Page!",`
`     "Bonjour de la Page d’accueil !",`
`      "Hola desde la página de bienvenida!"`
`    ]`

But as I have to share this file to multiple people for translations, I would be lot easier if I can create JSON file for different languages.
eg:
`locale_en.json`
   `"title": "Hello from Info Page!"`

`locale_fr.json`
   `"title": "Bonjour de la Page d’accueil !"`

`locale_es.json`
   `"title": "Hola desde la página de bienvenida!"`

**Comment by tsl143** [comment]
@ryandrewjohnson , FYI :)

**Comment by ryandrewjohnson** [comment]
I have been thinking about a way to provide some sort of transformation logic to be able to handle different json formats. I will give this some thought as I'd ideally like to come up with a way to allow the user to write their own transformations. 

I will keep this open and provide updates as things progress.

**Comment by ryandrewjohnson** [comment]
@tsl143 I have decided to develop the transformation idea as a separate feature instead, and will treat this as a it's own feature.  I was hoping you could help me out with testing...

I have pushed a new [feature branch ](https://github.com/ryandrewjohnson/react-localize-redux/tree/feature/add-translation-for-language) with this new functionality. If you could pull this branch and test it with your test case that would be great.

I've added a new action creator called `addTranslationForLanguage(translation, language)`. Using the example you provided you would now add your translation data like so...

```javascript
addTranslationForLanguage(require('locale_en.json'), 'en');
addTranslationForLanguage(require('locale_fr.json'), 'fr');
addTranslationForLanguage(require('locale_es.json'), 'es');
```
Once your translations are added everything else works the same, and the data looks exactly the same in the Redux store. The only caveat now is that you need to ensure that you have aded the translations for a language before activating it.

Let me know if you need any help getting the feature branch working.

**Comment by tsl143** [comment]
@ryandrewjohnson awesome, that was quick. I will test this asap and update here ASAP. Thanks :)

**Comment by ryandrewjohnson** [comment]
@tsl143 any luck with that testing? I'm just finishing up updating the docs on my end, and I'm hoping to push soon if all went well with your testing.

**Comment by ryandrewjohnson** [comment]
This feature is now released in `v2.5.1`.

**Comment by tsl143** [comment]
@ryandrewjohnson , sorry for late reply.
And yes this new feature is working super fine and smooth.
Take a bow sir, this library is the thing that fueled up my project :)
Once again many thanks for quick and awesome solution.
